spec/honeykiq/server_middleware_spec.rb

```diff
@@ -55,6 +60,24 @@ def perform(arg = nil)
     expect(honey_client.events.first.data).to include(expected_event)
   end
 
+  describe 'adding `extra_fields`' do
+    let(:reporter) { TestExtraFields.new(honey_client: honey_client) }
+    let(:expected_event_keys) { expected_event.keys << :extra_data_item }
+
+    before do
+      Sidekiq::Testing.server_middleware do |chain|
+        chain.add TestExtraFields, honey_client: honey_client
+      end
+    end
```

Comment:
1. I think `reporter` is not necessary, right? (I probably copy pasted from the other test file and you copy pasted from the above where you correctly deleted it. :P )
2. Doing another `before` block ends up chaining two middlewares. How about having a `let` which is used in the above `chain.add` to set the class that gets chained, that way you can override the `let` in this context. (Hopefully that made sense.)

Reply:
it's not, i deleted the other one and forgot that :|

that's a good idea, thanks